README.md

```diff
@@ -6,18 +6,30 @@
 The Jaeger Operator is an implementation of a [Kubernetes Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
 
 ## Getting started
-
-To install the operator, run:
+1) Create the namespace: 
 ```
 kubectl create namespace observability
+```
+2) Load the CRD based on your version:
+- Kubernetes 1.12+:
+```
+kubectl create -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/crds/jaegertracing.io_jaegers_crd.yaml
+```
+- Kubernetes 1.11-: 
```

Comment:
Possibly add  ` / OpenShift 3.11`?

Reply:
I refrained from talking about OCP beacuse it isn't mentioned anywhere here for some reason, I actually use Openshift personally,